Makefile

```diff
@@ -1,14 +1,31 @@
 
+GOBIN:=$(GOBIN)
```

Comment:
I wouldn't do that, or at least like that

GOBIN is a reserved Go environment variable.

It's default to GOPATH, and it's a folder, where binaries you install via `go install ` are installed.

You can see the path information by using `go env`

I would recommend choosing another variable

Reply:
you're right I spaced out on naming (naming is hard) - I use GO_BIN elsewhere let me fix